### PR TITLE
Add super simple header

### PR DESCRIPTION
**Pull request by reillz10**
@edx/educator-neem Check out my code

### CODE REVIEW

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <header className="d-flex justify-content-center align-items-center p-3 border-bottom-blue">
+          {/* <button className="bg-color-white border-0 p-0" onClick={() => this.setState({ mobileNavOpen: !this.state.mobileNavOpen })}>
+            <Icon
+              className={['fa', this.state.mobileNavOpen ? 'fa-times' : 'fa-bars', 'fa-2x', 'color-gray-dark']}
+            />
+          </button> */}
+          <Hyperlink content={<img src="./assets/edx-sm.png" alt="The edX logo" height="30" width="60" />} destination="https://www.edx.org" />
+          <div />
+        </header>
+        {/* {this.state.mobileNavOpen &&
```
Comment (iloveagent57) [inline src/components/Header/index.jsx]: You probably want to get rid of this comment block

File: src/components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    this.state = {
+      mobileNavOpen: false,
+    };
+  }
+
+  render() {
+    return (
+      <div className="mb-3">
+        <header className="d-flex justify-content-center align-items-center p-3 border-bottom-blue">
+          {/* <button className="bg-color-white border-0 p-0" onClick={() => this.setState({ mobileNavOpen: !this.state.mobileNavOpen })}>
```
Comment (iloveagent57) [inline src/components/Header/index.jsx]: Probably wanna get rid of this comment.

Comment (schenedx) [inline src/components/Header/index.jsx]: We should remove all these commented out code.

File: src/components/Header/header.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+.size-16 {
+    font-size: 16px;
+}
+
+.border-bottom-blue {
+    border-bottom: 1px solid #0075b4;
+}
+
+.border-bottom-gray {
```
Comment (iloveagent57) [inline src/components/Header/header.scss]: I think you're only using this for test code

File: src/components/Header/header.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.color-gray-dark {
+    color: #767676;
+}
+
+.weight-bold {
+    font-weight: 600;
+}
+
+.size-16 {
```
Comment (iloveagent57) [inline src/components/Header/header.scss]: only used for test code

File: src/components/Header/header.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
+.color-gray-dark {
+    color: #767676;
+}
+
+.weight-bold {
```
Comment (iloveagent57) [inline src/components/Header/header.scss]: only used for test code

File: src/components/Header/header.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
+.color-gray-dark {
```
Comment (iloveagent57) [inline src/components/Header/header.scss]: only used for test code

File: src/components/Header/header.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+.border-bottom-blue {
+    border-bottom: 1px solid #0075b4;
+}
+
+.border-bottom-gray {
+    border-bottom: 1px solid #e7e7e7;
+}
+
+.nav-link::after {
```
Comment (iloveagent57) [inline src/components/Header/header.scss]: only used for test code

**Pull request merged**